src/plugins/stepsform/stepsform.js

```diff
@@ -0,0 +1,163 @@
+				};
+			}
+
+			// remove submit button (user should add one for no script purposes)
+			$( eventTarget ).find( "input[type=submit]" ).remove();
+
+			// find all fieldsets, hide required content, and add appropriate buttons
+			var fieldsets = eventTarget.getElementsByTagName( "FIELDSET" ),
+				settings = wb.getData( eventTarget, componentName ),
+				form, fieldset, legend, div;
+			for ( var i = 0, len = fieldsets.length; i < len; i++ ) {
+
+				// only applies too fieldsets that have a parent element of form
+				fieldset = fieldsets[ i ];
+				form = fieldset.parentElement;
+				if ( form.nodeName.toUpperCase() === "FORM" ) {
+					var isFirstFieldset = ( i === 0 ) ? true : false,
+						isLastFieldset = ( ( i + 1 ) >= len ) ? true : false,
+						nextFieldsetParent = ( !isLastFieldset && fieldsets[ i + 1 ] ) ? fieldsets[ i + 1 ].parentElement : false,
+						previous, next, submit;
+
+					legend = fieldset.getElementsByTagName( "LEGEND" )[ 0 ];
+					div = fieldset.getElementsByTagName( "DIV" )[ 0 ];
+
+					// add previous step button
+					if ( !isFirstFieldset ) {
+						previous = document.createElement( "button" );
+						previous.innerHTML = i18nText.prv;
+						previous.id = componentName + "-prv-" + ( i + 1 );
+						previous.setAttribute( "class", "btn btn-md btn-default mrgn-rght-sm mrgn-bttm-md" );
+						previous.setAttribute( "type", "button" );
+						previous.setAttribute( "role", "button" );
+						previous.setAttribute( "aria-labelby", i18nText.prv );
+						previous.addEventListener( "click", function() {
+							displayFieldset( this.parentElement, false );
+						} );
+						div.appendChild( previous );
+					}
+
+					// add next step button or form submit button if last fieldset
+					if ( nextFieldsetParent && nextFieldsetParent.nodeName.toUpperCase() === "FORM" ) {
+						next = document.createElement( "button" );
+						next.innerHTML = i18nText.nxt;
+						next.id = componentName + "-nxt-" + ( i + 1 );
+						next.setAttribute( "class", "btn btn-md btn-primary mrgn-bttm-md" );
+						next.setAttribute( "type", "button" );
+						next.setAttribute( "role", "button" );
+						next.setAttribute( "aria-labelby", i18nText.nxt );
+						next.addEventListener( "click", function( evt ) {
+							evt.preventDefault(); // stop form submission
+
+							// confirm if form is valid
+							var isFormValid = true;
+							if ( jQuery.validator && jQuery.validator !== "undefined" ) {
+								isFormValid =  $( "#" + this.parentElement.parentElement.parentElement.id ).valid();
+								this.parentElement.previousElementSibling.setAttribute( "class", "wb-steps-error" );
+							}
+
+							// continue if valid
+							if ( isFormValid ) {
+								this.parentElement.previousElementSibling.removeAttribute( "class" );
+								displayFieldset( this.parentElement, true );
+							}
+
+						} );
+						div.appendChild( next );
+					} else {
+						var value = ( settings && settings.submitBtnVal ) ? settings.submitBtnVal : "X";
```

Comment:
- [ ] i18n Text (add "Submit" as default button text (enabling tmpl-submit (not sure about this)))